src/content/quickstart/verify_with_discovery.md

```diff
@@ -1,7 +1,12 @@
 #### Verify Deployment
 
-To verify the deployment follow the steps below which creates an nginx service and
-ServiceExport for it.
+To manually verify the deployment, follow the steps below using either a headless or ClusterIP `nginx` service deployed in `cluster-b`.
+
+```bash
+export KUBECONFIG=cluster-b/auth/kubeconfig
```

Comment:
I suggest including this in the sections below for consistency.  You already have it on L12 so add it before L21 as well.

Reply:
Done